openwisp_controller/config/base/vpn.py

```diff
@@ -772,6 +778,9 @@ class AbstractVpnClient(models.Model):
     # needed for wireguard
     public_key = models.CharField(blank=True, max_length=44)
     private_key = models.CharField(blank=True, max_length=44)
+    # needed for zerotier
+    member_id = models.CharField(blank=True, max_length=10)
```

Comment:
Do we really need to save `member_id` in the database? I think this duplicates the data in the database unnecessarily. 

We can create a cached property/cached_property for member_id 

```python
@cached_property
def member_id(self):
    return self.zt_identity_secret[:10]
```